docs/source/conf.py

```diff
@@ -54,6 +54,7 @@
 # a list of builtin themes.
 #
 html_theme = 'sphinx_rtd_theme'
+add_module_names = False
```

Comment:
What does this do and does it change anything for the other documentation?

Reply:
It prevents the entire module path from being displayed when using autodoc, and keeps it aligned with the rest of the documentation. It only affects autodoc statements.